external/fv3fit/fv3fit/_shared/models.py

```diff
@@ -437,3 +437,64 @@ def load(cls, path: str) -> "OutOfSampleModel":
 
         model = cls(base_model, novelty_detector, cutoff=cutoff, taper=taper)
         return model
+
+
+@io.register("squashed_output_model")
+class SquashedOutputModel(Predictor):
+
+    _CONFIG_FILENAME = "squashed_output_model.yaml"
+
+    def __init__(
+        self,
+        base_model: Predictor,
+        squash_by_name: Optional[Hashable],
```

Comment:
Since this model is manually constructed, it seems like there would not be a scenario where this is `None` and the prediction is just the base model prediction. Unless there's a case where you could imagine this, I'd suggest just making the type non optional and removing the if statement.